herd/unittests/AArch64/A128.litmus.expected

```diff
@@ -0,0 +1,10 @@
+Test A128 Required
+States 1
+0:X0=22; v=22; v+1=44;
+Ok
+Witnesses
+Positive: 1 Negative: 0
+Condition forall (0:X0=22 /\ v=22 /\ v+1=44)
```

Comment:
Something may have to change here, in the printing of the condition. The variable `v` being an array =, one expect to see `v[0]` and `v[1]` and not `v` and `v+1`.  Of course for `x` declared as a scalar `int x=0`, we do not want to have it printed as `v[0]`. Hence printing of conditions and outcome would depend upon types. This is manageable, I hope. I see no big difficulty for herd (just find the type in some environment) or litmus (which already relies much on types).  More so for tools such as mcompare...

Reply:
BTW: We already have a problem which could be solve by considering types while printing values. Ccnsider:
```
AArch64 T
{
int x=-1;
0:X1=x;
}
 P0          ;
LDR W0,[X1]  ;
locations [0:X0;]
````
Result is `0:X0=4294967295;`, which is correct in some sense. In fact the integer is printed as it is internally (`Int64.t`), while one should ignore the high order half and print it as a signed int (the defaut type). While litmus does correctly.